docs/Apps/Metabolomic Data/Labeled LC-MSMS Workflow.md

```diff
@@ -139,6 +141,86 @@ Click "Run* to plot the fractional enrichment plot with all the labels present a
 
 ![Visualization plot](../../img/LabeledLC-MS2Workflow/VisualizationPlot.png) <center> **Figure 16.** Visualization plot </center>
 
+## Phi (Beta) Tab
+
+This tab enables the user to perform Phi analysis. The term 'Phi' denoted by 'φ' is ratios of fractional enrichments of isotopomers/isotopologues of a product to the fractional enrichments of isotopomers/isotopologues of a precursor in the pathway.
+
+- Consider, ![A to B to C](../../img/LabeledLC-MS2Workflow/phi3.png), then ![Phi calculation formula](../../img/LabeledLC-MS2Workflow/phi2.png), which means if there is **another flux** contributing to the generation of B, then other pathway contributes to **FE**<sub>**B* **</sub> in this case, **𝛷AB < 1**
+
+- For eg: In the image shown below, Acetyl-CoA is generated from Pyruvate and with some contribution from Fatty acids. Then, **𝜱PAc** < **1** as Acetyl-CoA is also formed from Fatty Acids. If the only contributor is Pyruvate then **𝜱PAc** = **1**
+
+![Acetyl CoA is generated from Pyruvate and Fatty acids](../../img/LabeledLC-MS2Workflow/tca.png)<center>**Figure 20:** Acetyl CoA is generated from Pyruvate and Fatty acids</center>
+
+**1. Calculate Phi:** This allows the user to set the parameters to calculate the phi values based on the formula specified above. User can upload NA corrected file if it has been performed externally. In case, NA correction is performed within the app, there is no need to upload the NA corrected file and the user can directly perform Phi calculation.
```

Comment:
We don't follow numeric segregation. Can you change this to keep the entire format consistent

Reply:
@KMaheswari Also put beta in superscript similar to OCR Beta.